template/web/app/paymaster-bundler/_components/PaymasterBundlerDemo.tsx

```diff
@@ -1,174 +1,96 @@
+    } catch (error) {
+      console.error('Error writing contract: ', error);
+    }
+  };
 
-    void createSmartAccount();
-  }, [privyClient]);
+  const handleMintClick = () => {
+    handleMint().catch(error => {
+      console.error('Error in handleMint:', error);
+    });
+  };
 
   return (
-    <div className="mb-10 rounded-xl border border-boat-color-palette-line">
-      <Header />
-      <div className="relative lg:flex">
-        <Vault ownedTokens={ownedTokens} />
-        <GamePlay
-          setOwnedTokens={setOwnedTokens}
-          setShowSponsoredModal={setShowSponsoredModal}
-          setTransactionHash={setTransactionHash}
-          setMintedItem={setMintedItem}
-          smartAccount={smartAccount}
-          client={client}
-        />
-        {showSponsoredModal && (
-          <div className="absolute inset-0 flex items-center justify-center">
-            <div className="w-700 h-700 relative z-10 rounded-2xl border border-gray-600 bg-black p-6 shadow-lg">
-              <button
-                type="button"
-                className="absolute right-3 top-2 text-white"
-                onClick={() => setShowSponsoredModal(false)}
-              >
-                X
-              </button>
-              <div className="mb-5 flex flex-col items-center">
-                <Item
-                  src={ALL_ITEMS[mintedItem].image}
-                  altText={ALL_ITEMS[mintedItem].name}
-                  className="mx-auto my-auto mb-5 max-h-[100px] max-w-[100px]"
-                />
-                <p>{ALL_ITEMS[mintedItem].name}</p>
-              </div>
-              <div className="flex flex-col items-center">
-                <p className="text-sm text-green-400">NFT mint successfully sponsored!</p>
-                <p className="text-sm">{ALL_ITEMS[mintedItem].name} added to your vault.</p>
-                <button
-                  type="button"
-                  className="mt-5 block w-full rounded-full border border-boat-color-orange py-4 hover:bg-gray-800"
-                  onClick={() => window.open(`https://sepolia.basescan.org/tx/${transactionHash}`)}
-                >
-                  <div className="flex justify-center">
-                    <span>View on Basescan</span>
-                    <img
-                      src="/account_abstraction/etherscan.png"
-                      alt="Etherscan"
-                      className="mb-1 ml-2 mt-1 h-5 w-5"
-                    />
-                  </div>
-                </button>
-              </div>
-            </div>
+    <div className="p-5 font-sans">
+      <div className="mb-5">
+        <h2 className="border-b-2 border-gray-300">Account Details</h2>
+        <div className="mt-2 text-lg">
+          <strong>Status:</strong> {account.status}
+          <br />
+          <strong>Chain ID:</strong> {account.chainId}
+          <br />
+          <strong>Addresses:</strong> {JSON.stringify(account.addresses)}
```

Comment:
Addresses is already at the top right of the page, we don't need this again. This seems a not need it copy and pasty from the Wagmi Smart Wallet demo.

Reply:
removed